src/screens/Settings/components/WineSettings/index.tsx

```diff
@@ -262,23 +262,26 @@ export default function WineSettings({
         }
         value={wineVersion.name}
         afterSelect={
-          <InfoBox text="infobox.help">
-            <span>{t('help.wine.part1')}</span>
-            <ul>
-              <i>
-                <li>~/.config/heroic/tools/wine</li>
-                <li>~/.config/heroic/tools/proton</li>
-                <li>~/.steam/root/compatibilitytools.d</li>
-                <li>~/.steam/steamapps/common</li>
-                <li>~/.local/share/lutris/runners/wine</li>
-                <li>~/.var/app/com.valvesoftware.Steam (Steam Flatpak)</li>
-                <li>/usr/share/steam</li>
-                <li>Everywhere on the system (CrossOver Mac)</li>
-                <li>/opt/cxoffice (CrossOver Linux)</li>
-              </i>
-            </ul>
-            <span>{t('help.wine.part2')}</span>
-          </InfoBox>
+          <>
+            <InfoBox text="infobox.wine-path">{wineVersion.bin}</InfoBox>
```

Comment:
Question: why did you prefer to add it to the help box?
I wonder if adding it like we are doing with the Legendary alternative binary or on the language selector, that small text below the input/select. I think its easier to see.

Reply:
While it would be easier to see, I thought this was something not necessary normally (as normally one would have only one installation of everything) and therefore wanted to hide it away only for the users who are confused with multiple versions of the same name.